github/search.go

```diff
@@ -200,6 +241,10 @@ func (s *SearchService) search(ctx context.Context, searchType string, query str
 		// Accept header for search repositories based on topics preview endpoint
 		// TODO: remove custom Accept header when this API fully launches.
 		req.Header.Set("Accept", mediaTypeTopicsPreview)
+	case searchType == "labels":
+		// Accept header for search labels based on label description preview endpoint
```

Comment:
nit: add period to end of comment. I see that the other comments don't, but Go style typically uses complete sentences for comments.